com.unity.multiplayer.mlapi/Runtime/Spawning/NetworkSpawnManager.cs

```diff
@@ -541,19 +541,28 @@ internal static void ServerResetShudownStateForSceneObjects()
             }
         }
 
+        /// <summary>
+        /// Gets called only by NetworkSceneManager.SwitchScene
+        /// </summary>
         internal static void ServerDestroySpawnedSceneObjects()
         {
-            foreach (var sobj in SpawnedObjectsList)
+            //This Allocation is "ok" for now because this code only executes when a new scene is switched to
```

Comment:
Is this comment still real?  I don't see an allocation per se...or do you mean the ToList() call?

Reply:
Yes ToList creates a copy of the collection and allocates a new list object in the process.